draft-ietf-quic-multipath.md

```diff
@@ -690,10 +692,10 @@ for 1-RTT packets requires changes in AEAD usage.
 the use of a nonce, N, formed by combining the packet protection IV
 with the packet number. When multiple packet number spaces are used,
 the packet number alone would not guarantee the uniqueness of the nonce.
-
-In order to guarantee the uniqueness of the nonce, the nonce N is
-calculated by combining the packet protection IV with the packet number
-and with the least significant 32 bits of the Path ID.
+Therefore, the nonce N is calculated by combining the packet protection
+IV with the packet number and with the least significant 32 bits of the
+Path ID. In order to guarantee the uniqueness of the nonce, the Path ID
+is limited to a max value of 2^30-1.
 
```

Comment:
Should be "is limited to a max value of 2^32-1." We want 32 bits, not 30 bits. But this is redundant with what we say in the section "Introduction of an Explicit Path Identifier." We should not have two places stating the limit.

Reply:
ups, yes I definitely messed this up here. I'll fix it to the right value now and then make another proposal to repeat this less often.